src/docs/concept/kubefile.md

```diff
@@ -72,9 +72,13 @@ USAGE：
 
 For example:
 
-+ Using `APP` instruction to define a `mysql` app and it will use remote chart file `https://charts/mysq.tgz` to start it and the remote file will be auto downloaded.
++ Using `APP` instruction to define a `mysql` app and it will use local chart dir `https://charts/mysql/` to start it.
```

Comment:
Using `APP` instruction to define a `mysql` app and it will use local chart dir `local://mysql` to start it.